install/first_google_boot.sh

```diff
@@ -249,26 +249,43 @@ function extract_spinnaker_gcr_credentials() {
   mkdir -p $(dirname $config_path)
   chown -R spinnaker:spinnaker $(dirname $config_path)
 
+  local gcr_enabled=$(get_instance_metadata_attribute "gcr_enabled")
   local gcr_account=$(get_instance_metadata_attribute "gcr_account")
-  if [ -n "$gcr_account" ]; then
-    # This downloads the JSON key for the specified service account
+
+  if [ -n "$gcr_enabled" ] || [ -n "$gcr_account" ]; then
+    echo "GCR enabled"
+
```

Comment:
How unusual would it be to set a gcr_account but not enable it?
I assume this || is to go the other way around and use the default account.
Could you use an empty account to imply the default and gcr_enabled = ! [-z gcr_account] ? Just asking.